dist/dotty.rb

```diff
@@ -0,0 +1,32 @@
+class Dotty < Formula
+  desc "Experimental Scala Compiler"
+  homepage "http://dotty.epfl.ch/"
+  url "http://dotty.epfl.ch/dotty/dotty-0.1.1-bin-SNAPSHOT.tar.gz"
+  sha256 "4e1bda148754543844d25290a87076e6bfb0b6b0275535f97c1871e0fc5c2c4c"
```

Comment:
If you upload the artifacts to Github releases, then the sha will be different than running sha locally. You would have to upload, download and then run sha on the dowloaded file.